Resources/ServerInfo/Guidebook/NewPlayer/Controls/Radio.xml

```diff
@@ -0,0 +1,65 @@
+<Document>
+# Speech and Text Channels
+Talking is a key part of this game, both literally and fuctionally. You can press "T" to jump to the text box.
```

Comment:
There's a keybind markup tag that can be used to pull the actual key the user has bound to an action. Try to use it whenever possible so people can change their controls and still have an accurate guidebook!